assets/script.js

```diff
@@ -1,7 +1,9 @@
-
-function opneNave (){
-    document.getElementById("mobileNave").style.width = "100%";
+function opneNave() {
+  document.getElementById('mobileNave').style.width = '100%';
+}
+function closeNave() {
+  document.getElementById('mobileNave').style.width = '0';
```

Comment:
- [ ] Please try to use `Javascript` *best practices* by using the **arrow** function instead of the old js function, because it will make your code with current IT industry standards and also help you to understand things in the better and more familiar manner. Don't worry soon you will come to learn more about things regarding Javascript. Kindly have a look at the below code snippet.

```suggestion
const opneNave = () => {
  document.getElementById('mobileNave').style.width = '100%';
}
const closeNave = () => {
  document.getElementById('mobileNave').style.width = '0';
```